boefjes/boefjes/config.py

```diff
@@ -68,6 +80,12 @@ class Settings(BaseSettings):
         validation_alias="KATALOGUS_DB_URI",
     )
 
+    db_engine_pool_size: int = Field(
+        25,
+        description="Pool size for create engine.",
+        validation_alias="DB_ENGINE_POOL_SIZE",
```

Comment:
"for create engine" doesn't really describe what this does. I think it should also be possible to specify this per service instead of a global `DB_ENGINE_POOL_SIZE`

```suggestion
    db_connection_pool_size: int = Field(
        25,
        description="Database connection pool size",
```